src/components/Scaler/index.tsx

```diff
@@ -1,123 +1,125 @@
 import * as React from 'react';
-import * as PropTypes from 'prop-types';
-import DataContext from '../../context/Data';
```

Comment:
We should remove unused `../../context/Data`

Reply:
Good catch!